styles/modules/modals/_orderDetail.scss

```diff
@@ -1,5 +1,223 @@
 .orderDetail {
+  .returnBox {
+    max-width: 905px;
+    text-decoration: none;
+    padding: $padSm;
+
+    .avatar {
+      width: 24px;
+      height: 24px;
+      float: left;
+    }
+  }
+
+  .featuredProfile {
+    text-align: center;
+    padding: $padMd;
+    min-height: 143px;
+
+    .avatar {
+      width: 70px;
+      height: 70px;
+      margin: 0 auto 7px;
+    }
+  }
+
+  .mainCtaWrap {
+    .btn {
+      width: 100%;
+    }
+  }
+
+  .tabHeads {
+    .unreadBadge {
+      display: inline-block;
+      width: 20px;
+      height: 20px;
+      text-align: center;
+      font-weight: bold;
+      padding-top: 3px;
+      font-size: $tx6;
+      margin-left: 5px;
+
+      &:empty {
+        opacity: 0;
+        // needed so position of the badge is consistent with the non-empty
+        // version and there's no jump as you toggle between the two
+        vertical-align: top;
+      }
+    }
+  }
+
   .mainContent {
     min-height: 300px;
   }
+
+  .summaryTab, .contractTab {
+    padding: $padMd;
+  }
+
+  .discussionTab {
+    position: relative;
+    overflow: hidden;
+
+    &.loadingMessages {
+      .convoMessagesWindow {
+        .spinner {
+          display: block;
+          width: 55px;
+          height: 55px;
+        }
+      }
+
+      &.noMessages {
+        .convoMessagesWindow {
+          .spinner {
+            @include center();
+          }
+
+          .messagesFetchError {
+            @include center();
+          }
+        }
+      }
+    }
+
+    &.noMessages {
+      .convoMessagesWindow {
+        .messagesFetchError {
+          @include center();
+          margin: 0;
+        }
+      }
+    }
+
+    &.isTyping {
+      .typingIndicator {
+        transform: translate(-50%, 0);
+      }
+    }
+
+    .convoMessagesWindow {
+      padding: $padLg;
+      overflow-x: hidden;
+      overflow-y: auto;
+      max-height: 375px;
+      min-height: 375px;
```

Comment:
Yeah, I'll switch it to just height. The min / max height approach was needed on tables since the table only occasionally honored the non min / max value... but that's probably a tables only quirk.